### DISCUSSION TITLE
New ultimate battery care function

### DISCUSSION THREAD

**Discussion by EricChenSudo** [seerge/g-helper#2755] [Ideas]
Dear seerge,

I hope this message finds you well. I have a question regarding the battery limit settings. Previously, someone mentioned the possibility of implementing a feature similar to myASUS, where the laptop, when plugged in, allows the battery to drop to 78% or 58% before starting to charge back up to 80% or 60%. I noticed that your response at the time indicated that we can only control the BIOS maximum limit.

Here’s an idea: what if we add an option called "Battery Care" next to the battery limitation control? If checked, when the battery reaches the set battery limitation, Ghelper could adjust the BIOS battery limitation to 5% below the user-defined battery limitation. It would remain at this setting until the battery naturally drops to 75% or 55%, at which point Ghelper would reset the BIOS battery limitation back to the original user-defined setting.

I believe this could enable Ghelper to provide battery protection similar to myASUS, or even better, since myASUS can only set 60% and 80%, whereas Ghelper could offer more customizable settings. This is how I am currently managing my battery manually, so I think it is a feasible approach.

Thank you once again for creating such a fantastic program!

**Comment by seerge** [comment]
@EricChenSudo hello, 

MyASUS (or actually Asus Optimization Service) does exactly same as G-Helper. It runs a single command telling firmware to hold specific charge - 60 or 80 in that case. It doesn't do anything else besides that :)

**Comment by Tcpprkg** [comment]
nice idea
